packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -122,9 +122,22 @@ String msText(
   Duration dur, {
   bool includeUnit = true,
   int fractionDigits = 1,
+  bool allowZeroValues = true,
```

Comment:
Can you add documentation to this method that will explain the method itself behavior in case of zeros?

Reply:
done